functions/src/strings.js

```diff
@@ -151,13 +151,20 @@ module.exports = {
       /**
        * When user missed the available range
        * we should help them to find alternative.
+       *
+       * Hints:
+       * - gets group of speeches with the most intersection with existing slots
+       *   and get random
+       * - doesn't match to empty suggestions
        */
       repair: {
         speech: [
           `I don’t have anything for {{year}}. Try {{suggestions.0}}, for example.`,
           `I don't have {{creator}} albums for {{year}}. Try {{suggestions.0}}, for example.`,
           `I don't have any albums for {{year}}. Try {{suggestions.0}}, for example.`,
           `I don't have that. Try {{suggestions.0}}, for example.`,
+          `I don't have {{subject}} of {{year}}. Try {{suggestions.0}}, for example.`,
+          `I don't have {{subject}} of {{year}}. Maybe you would like to listen something else?`,
```

Comment:
The grammar here is a little clunky. I would change it from `I don't have {{subject}} of {{year}}` to `I don't have {{subject}} for {{year}}`

Reply:
Same thing goes for the previous instance in the line above this one.